pkg/sql/codegen/pai/codegen.go

```diff
@@ -32,10 +32,12 @@ import (
 const (
 	// ModelTypeTF is the mode type that trained by PAI Tensorflow.
 	ModelTypeTF = iota
-	// ModelTypeRandomForests is the model type that trained by PAI random forests.
-	ModelTypeRandomForests
 	// ModelTypeXGBoost is the model type that use PAI Tensorflow to train XGBoost models.
 	ModelTypeXGBoost
+	// ModelTypeKMeans is the model type that trained by PAI KMeans
+	ModelTypeKMeans
+	// ModelTypePAIML is the model type that trained by PAI machine learing algorithm tookit
+	ModelTypePAIML
```

Comment:
Keep the name `ModelTypeRandomForests`? since `ModelTypeKMeans` is also using PAI command only to run

Reply:
Maybe `ModelTypeRandomForests` is not exactly, `KMeans` and `RandomForests` can also use this model type, how about `ModelTypeOthers` ?